tap_mixpanel/__init__.py

```diff
@@ -71,11 +70,19 @@ def main():
     else:
         api_domain = "mixpanel.com"
 
+    auth_type = parsed_args.config.get("auth_type")
+    if not auth_type:
+        auth_type = "api_secret"
```

Comment:
With current implementation `auth_type='<whitespace>` is an acceptable value, should be defaulted to default_value.

```suggestion
    if not (auth_type and auth_type.strip()):
        auth_type = "api_secret"
```

Reply:
```python
    # default to api_secret as authentication_type
    if auth_type not in ("saa","api_secret"):
        auth_type = "api_secret"
```